docs/src/components/tag-badge.tsx

```diff
@@ -0,0 +1,30 @@
+import type { Tag } from "@/lib/get-tags";
+import { cn } from "@/lib/utils";
+import type { ReactNode } from "react";
+import { Badge } from "./ui/badge";
+
+const TAG_COLOR = {
+  Lazy: "bg-green-600 text-green-50 hover:bg-green-600/80 dark:bg-green-500 dark:text-green-950",
+
+  Indexed:
+    "bg-purple-600 text-purple-50 hover:bg-purple-600/80 dark:bg-purple-500 dark:text-purple-950",
+
+  Strict:
+    "bg-rose-600 text-rose-50 hover:bg-rose-600/80 dark:bg-rose-500 dark:text-rose-950",
+} as const satisfies Readonly<Record<Tag, string>>;
```

Comment:
**thought:** this is one of the things i dislike about the way we compile tailwind... you'd think we could just do a variable with the color, but that doesn't work!

Reply:
I don't think these colors are even very good, but because in v2 we only have one tag, I wouldn't put too much effort into optimizing this. The result is cute and makes sense, and this is the easiest way to do it.